Header header
int64  frame_id
int8   command
sensor_msgs/Image img0
sensor_msgs/Image img1
int32 lm_count
std_msgs/Int64MultiArray lm_id_data
geometry_msgs/Vector3[]  lm_2d_data
geometry_msgs/Vector3[]  lm_3d_data
std_msgs/UInt8MultiArray lm_descriptor_data
geometry_msgs/Transform  T_c_w

